### PR TITLE
PALT PR Template

### PR DESCRIPTION
**Pull request by landonarogers**
## Type

- Update to Git files

## Problem

There is no standardized form of PR documentation.

## Solution

Create a PR template, so that all future PR's will start with the same generic format.

## Changes

- Created a pull_request_template.md file (pr template)

## Validation

- Compared to other PR Templates
- Talked to other team members

### CODE REVIEW

**Comment by NavneetThekkumpat** [comment]
Would be nice to have a validation section too

**Comment by landonarogers** [comment]
Template example as PR description

**Comment by arpavan** [comment]
Also specify that if the PR fixes an issue, link to that issue in GitHub, and if it doesn't exist, create a GitHub issue and link it. That way there's a paper trail to the original motivation for this fix.

**Comment by landonarogers** [comment]
> Also specify that if the PR fixes an issue, link to that issue in GitHub, and if it doesn't exist, create a GitHub issue and link it. That way there's a paper trail to the original motivation for this fix.

Does every PR need an issue attached to it? If one doesn't already exist, do we need to make one? I'm wondering if making an issue just for a PR that we're already creating, which also explains the problem/solution, might be redundant and extra work?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Type
+
+Only keep relevant type. Delete the rest and this comment. If this is related to an issue open in GitHub, please link it to this ticket and put the URL here.
```
Comment (arpavan) [inline .github/pull_request_template.md:3]: Wasn't there a suggestion to do away with the categories?

**Review by arpavan — APPROVED** [review]

**Pull request merged**